src/constraint_system/composer.rs

```diff
@@ -22,15 +22,13 @@ use crate::constraint_system::Variable;
 use crate::permutation::Permutation;
 use ark_ec::models::TEModelParameters;
 use ark_ec::PairingEngine;
+#[cfg(feature = "trace")]
 use ark_ff::{BigInteger, PrimeField};
```

Comment:
I kept this on its own line since it uses a `feature` but I didn't check if we are doing this uniformly in the library, so either way is fine for now. Just letting you know what my preference is.

Reply:
Well, it's `rustfmt` the tool which merges the deps even if they have feature flags. 

Therefore I assume is expected to be on this way.

Reply:
Yeah, that's why I separate them by a newline so that `rustfmt` doesn't merge them. It's just a personal preference but let's keep it the way you have it now.

Reply:
Thanks! Let's see if we end up having more cases we will be able to judge better.